src/routing.rs

```diff
@@ -372,9 +343,13 @@ where
 
 #[cfg(test)]
 mod tests {
+    use wasm_bindgen_test::*;
+
     use super::*;
 
-    #[test]
```

Comment:
Is it possible to leave this annotation in as well so that both `test` and `wasm_bindgen_test` are available? Just curious.

Reply:
With both:
- `cargo make test_h_firefox` seems to work.
- `cargo test` seems to run test successfully, then fails on some errors

So it seems to work at least for tests which doesn't need browser to run.